### PR TITLE
Added addition wait to fix the flaky test for change language

### LINKED ISSUE #22566: Fix "Settings - general tab, validate the change language functionality" flaky tests
Flaky tests include:

- Settings - general tab, validate the change language functionality: User selects "Español" language and verify that changing the language from the default
- Settings - general tab, validate the change language functionality: User selects "Español" language from the dropdown and change back to english and verify that the word is correctly changed back to english
- Settings - general tab, validate the change language functionality: User selects "Dansk" language and verify that navigating to a different page does not affect
- Settings - general tab, validate the change language functionality: User selects "Magyar" language and verify that hypertext are updated with the selected language change
- Settings - general tab, validate the change language functionality: User selects "العربية" language and verify that page indent with the selected language change

### PR DESCRIPTION
**Pull request by hjetpoluru**
## **Description**
Tries to fix the "Settings - general tab, validate the change language functionality" flaky tests by adding waits
Fixes: #22566

### CODE REVIEW

**Review by HowardBraham — DISMISSED** [review]

File: test/e2e/tests/change-language.spec.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ describe('Settings - general tab, validate the change language functionality:',
         );
         assert.equal(isLanguageLabelChanged, true, 'Language did not change');
```
Comment (danjm) [inline test/e2e/tests/change-language.spec.js:86]: We are trying to move await from using delays in our tests. I wonder what this test needs to wait for before is correctly selected?

Comment (HowardBraham) [reply test/e2e/tests/change-language.spec.js:86]: In general, yes, but I think it's going to require some large structural changes to driver.js and helpers.js to accomplish this goal.

Comment (hjetpoluru) [reply test/e2e/tests/change-language.spec.js:86]: Sure I agree. I will make the changes to wait for the element.

**Review by danjm — CHANGES_REQUESTED** [review]
See my comment for the suggested change

File: test/e2e/tests/change-language.spec.js (hunk 1 above)
Comment (hjetpoluru) [reply test/e2e/tests/change-language.spec.js:86]: Done

**Review by hjetpoluru — COMMENTED** [review]
I added a wait for the spinner to finish loading before clicking the dropdown element.

**Review by danjm — APPROVED** [review]

**Review by legobeat — APPROVED** [review]

**Pull request merged**